src/explorer/editors/setAppWizardContextDefault.ts

```diff
@@ -0,0 +1,66 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Did you mean to put this in "editors"?

Reply:
No... "explorer". That seems like the appropriate spot, right?